docs/_includes/toggle-button.html

```diff
@@ -0,0 +1,254 @@
+<div id="toggle-button">
+    {% include section-header.html name="toggle-button" version=page.versions.toggle-button %}
+
+    <p>The Toggle Button is a button that can be toggled/pressed on or off. It is an atomic piece of the Toggle Button Group. Titles are required and other elements, such as subtitles and images can be included as well.</p>
+
+    <p>For detailed requirements and usage details, please review the <a href="https://ebay.gitbook.io/mindpatterns/input/toggle-button">eBay MIND Pattern for Toggle Button</a></p>
+
+    <p>To use these buttons as a set of multi-selection or single-selection variations, use a <span class="highlight">&lt;ul aria-label="Single-selection of payment options"&gt;</span> with buttons inside <span class="highlight">&lt;li&gt;</span>.</p>
```

Comment:
I think we can delete this line, or simply provide a link to Toggle Button Group when we have it.